test/templates/modal/component.njk

```diff
@@ -1,45 +1,26 @@
-{% import "../forms.njk" as forms %}
-{% from "../button/component.njk" import button %}
-{% from "../icon/component.njk" import icon %}
 <!--
 modal:
     properties={},
 -->
 {% macro modal(properties={}) %}
 <div class="fd-modal" role="dialog">
-  <div class="fd-modal__content" role="document">
-      <header class="fd-modal__header">
-      {% set title_size = properties.header_size  if properties.header_size  else "h3" %}
-      <{{title_size}} class="fd-modal__title">{{ properties.header }}</{{title_size}}>
-          {{  button(
-                  {
-                      label: item.label
-                  },
-                  modifier={
-                      block: ['light']
-                  },
-                  classes=['modal__close'])
-          }}
-      </header>
-      <div class="fd-modal__body">
+    <div class="fd-modal__content" role="document">
+        <header class="fd-modal__header">
+            {% set title_size = properties.header_size if properties.header_size else "h3" %}
+            <{{title_size}} class="fd-modal__title">{{ properties.header }}</{{title_size}}>
+            <button class="fd-modal__close"></button>
```

Comment:
Primary changes are using basic `<button>`s (this one has a class for the icon).

Reply:
There's a Slack thread with Greg debating whether `.fd-modal__close` (the modal CSS) should bring in the icons for the close "X" icon, or if we can remove that and leave it with the button CSS, trusting the component libraries to apply the right icon class to the button.

I could go either way. @jbadan do you have a leaning one way or the other?

Reply:
No real preference, I'd rather keep these as simple as possible so I'm more leaning towards removing it and trusting the implementation libraries to know what to do.

Reply:
I think there is enough chatter about this and no real detractors so let's go ahead and make this change.  We are version 0, so let's do this. 😄  This will allow to remove icons from modal self-contained styles.  We will keep the `fd-modal__close` class, but it will just handle the positioning of its child (button).  The implementation libraries will then be using their respective button component within the modal and will supply the proper icon to it.

Reply:
Excellent - this also just came up in the `tree` component I'm working on. I'll follow this pattern ✅